### ISSUE TITLE
File upload field

### ISSUE THREAD

**Issue by lukecarbis** [getblocklab/block-lab#281]
Upload any file (or select an existing one). Choose the file type from the control settings.

**Comment by hybridherbst** [comment]
Just found out about Block Lab and really like the idea, but without the ability to have files as custom fields it doesn't really make sense to use. Main usecase would be making blocks for specific types of file embeds for us.

Are there any "easy" workarounds to this? E.g. could I manually upload files to ftp and link them as plain text fields?
Any idea when/if that will happen?

**Comment by lukecarbis** [comment]
@soraryu Yes, the workaround is to upload the file somewhere (even your media library), and then use a text field (or even better, a URL field) to store the download URL.
